pyscitt/pyscitt/key_vault_sign_client.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+import base64
+import hashlib
+
+from azure.identity import DefaultAzureCredential
+from azure.keyvault.certificates import CertificateClient, KeyVaultCertificate
+from azure.keyvault.keys import KeyClient
+from azure.keyvault.keys.crypto import CryptographyClient, SignatureAlgorithm
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric.utils import encode_dss_signature
+from cryptography.x509 import load_pem_x509_certificate
+
+from . import crypto
+
+
+class KeyVaultSignClient:
```

Comment:
```suggestion
class KeyVaultSignClient(MemberAuthenticationMethod):
```

You should inherit from the abstract class.